modules/tinymce/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 - The Autolink plugin did not work when the text nodes in the content were fragmented #TINY-3723
 - Fixed various incorrect types on public APIs found while enabling TypeScript strict mode #TINY-8806
+- Text returned from `editor.getContent({format: 'text'})` would differ between some browsers #TINY-8579
```

Comment:
```suggestion
- Text returned from `editor.getContent({format: 'text'})` would differ with blank lines between some browsers #TINY-8579
```
Maybe to just make it a little clearer as to how it would differ?